### DISCUSSION TITLE
Add ping to transmit to keep connection open

### DISCUSSION THREAD

**Discussion by adrianNEMO** [adonisjs/core#4423] [Ideas]
It would be nice for transmit to send a periodic ping message to prevent the connection to timeout when using services like Cloudflare which closes connections after 100 seconds when no data is sent.
As a workaround I created a adonis provider which sends a event periodically to all clients. But this also requires to explicitly subscribe on the channel client.
It would be nice if transmit would send by default a message to all clients which can be just ignored on the client side. Since it does not require a subscription

**Comment by RomainLanz** [comment]
Hey @adrianNEMO! 👋🏻 

Great addition. So, the server should send a special event to the client that will be automatically dismissed.
This can be added as an option.

Can you create an issue in Transmit repository?

**Reply by adrianNEMO** [reply]
Correct I am going to add an issue
